src/main/features/dashboard/helpers/claimStatusFlow.ts

```diff
@@ -0,0 +1,49 @@
+import { Claim } from 'claims/models/claim'
+import { Logger } from '@hmcts/nodejs-logging'
+
+const logger = Logger.getLogger('ClaimStatusFlow')
+
+export class ClaimStatusFlow {
+
+  static readonly flow: ClaimStatusNode = {
+    description: 'Claim Is Draft',
```

Comment:
@andrewwa-kainos made me aware that this is not really true (as drafts are handled in a different way compared to claims), so this description is not correct. It should be more like `Claim exists`, it's just a root state which kicks off the whole flow.  I'll change that when I have to address the review comments which will follow :D